src/main/java/org/example/P04_TakingTwoNumbersWithScannerPlusTheirSum.java

```diff
@@ -0,0 +1,15 @@
+package org.example;
+import java.util.Scanner;
+
+
+public class P04_TakingTwoNumbersWithScannerPlusTheirSum {
```

Comment:
Rename to P04_SumTwoNumbers. You don't need to describe the details of the implementation, that you used a scanner. Also, the word 'Plus' in the name overlaps with the requirements and is misleading.